notebooks/official/pipelines/metrics_viz_run_compare_kfp.ipynb

```diff
@@ -2,13 +2,13 @@
   "cells": [
```

Comment:
<p>Remove 'Skip this step'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>